packages/wc-client/src/index.ts

```diff
@@ -412,11 +458,28 @@ export class KeplrWalletConnectV1 implements Keplr {
     return new CosmJSOfflineSignerOnlyAmino(chainId, this);
   }
 
-  getSecret20ViewingKey(
-    _chainId: string,
-    _contractAddress: string
+  async getSecret20ViewingKey(
+    chainId: string,
+    contractAddress: string
   ): Promise<string> {
-    throw new Error("Not yet implemented");
+    return (
+      await this.sendCustomRequest({
+        id: payloadId(),
+        jsonrpc: "2.0",
+        method: "keplr_get_scrt20_viewing_key_wallet_connect_v1",
+        params: [chainId, contractAddress],
+      })
+    )[0];
+  }
+
+  async getSecret20ViewingKeyOrPermit(
+    chainId: string,
+    contractAddress: string
+  ): Promise<{ permit: any | undefined; viewing_key: string | undefined }> {
+    return {
```

Comment:
Probably add todo comment here that once permits are implemented they should be added here.